dcpy/connectors/edm/recipes.py

```diff
@@ -257,11 +259,23 @@ def import_dataset(
             pg_dump_table_name=ds.id,
             target_table_name=ds_table_name,
         )
+
+        if rename_pk:
```

Comment:
This is needed if importing two versions of the same dataset using pg_dump, which I am doing while comparing datasets. Little bit of an odd case

Reply:
Should we move this to `import_pg_dump`, and always rename the pk if we specify `target_table_name`?

Reply:
that makes sense to me